task/build-image-index/0.1/build-image-index.yaml

```diff
@@ -147,3 +150,59 @@ spec:
       capabilities:
         add:
           - SETFCAP
+
+  - image: quay.io/redhat-appstudio/sbom-utility-scripts-image:d4935b35eb0849841412d5d1e481a812e939f2bf
```

Comment:
Please use the same image reference as other tasks: `quay.io/redhat-appstudio/sbom-utility-scripts-image@sha256:53a3041dff341b7fd1765b9cc2c324625d19e804b2eaff10a6e6d9dcdbde3a91`

If it doesn't include the code you need, you can use a newer version but please pin the image to a digest